hack/images/titus-sshd/Dockerfile

```diff
@@ -3,35 +3,48 @@ RUN emerge-webrsync
 RUN emerge --sync
 RUN echo 'MAKEOPTS="-j32"' >> /etc/portage/make.conf
 RUN echo 'USE="-pam -gdbm -berkdb -bindist"' >> /etc/portage/make.conf
-# This is because Docker builds don't have CAP_SYS_PTRACE
-RUN echo 'FEATURES="-sandbox -usersandbox"' >> /etc/portage/make.conf
-RUN echo "=sys-auth/libnss-compat-1.2 ~amd64" >> /etc/portage/package.accept_keywords
+# This is because Docker builds don't have CAP_SYS_PTRACE and cannot unshare
+RUN echo 'FEATURES="-sandbox -usersandbox -ipc-sandbox -network-sandbox -pid-sandbox"' >> /etc/portage/make.conf
```

Comment:
It probably makes sense to combine all these writes to make.conf?

Reply:
Eh, I would like to just append the stuff we want and take the rest of make.conf... from stock?